src/localization/locale/en.ts

```diff
@@ -86,6 +86,15 @@ export const en = {
     updateWorkQueue: 'Work queue updated',
   },
   info: {
+    today: 'today',
+    yesterday: 'yesterday',
+    tomorrow: 'tomorrow',
+    artifactLabel: 'Artifact',
+    artifactResultLabel: 'Result',
+    artifactCreatedTimestampLabel: 'Created',
+    artifactUpdatedTimestampLabel: 'Last Updated',
+    artifactRunLabel: 'Run',
+    artifactRunNone: 'None',
```

Comment:
I'd shorten these to basically what text they are. Remove the artifact prefix and label suffix. We can use these for things other than artifacts and label is a little redundant IMO